tests/console/zfs.pm

```diff
@@ -102,7 +102,8 @@ sub cleanup {
 
 sub scrub {
     my $pool = shift;
-    assert_script_run("zpool scrub $pool", timeout => 180);
+    # We may need retry several times in case the zpool is currently resilvering.
+    script_retry("zpool scrub $pool", delay => 60, retry => 3, timeout => 180);
```

Comment:
This is suboptimal, because like this we miss all other error cases. Here it might be better to check if the pool is already being scrubbed and in such case don't start a new scrub

```suggestion
    my $in_progress = script_run("zpool status $pool | grep -e 'scan:.*in progress'") == 0;
    assert_script_run("zpool scrub $pool", timeout => 180) if (!$in_progress);
```

Reply:
Thanks much!